data/paver/skeleton/templates/__init__.py.template

```diff
@@ -1,8 +1,16 @@
-from __future__ import unicode_literals
-
-{% if webapp %}import cherrypy
+from __future__ import unicode_literals{% if django %}
```

Comment:
One of the things we added was "Django support".  I put that in quotes because Django was always supported in the sense that you could use it, but now we have an actual ``--django`` flag in ``paver create_plugin`` and a section in our tutorial which explains how to most easily add a Django project to your Sideboard app.